salt/tests/unit/modules/test_cri.py

```diff
@@ -180,15 +180,51 @@ def _cmd_run_all_mock(cmd):
 
     @parameterized.expand(
         [
+            # Success: Found one container
             (None, 6, 0, "292c3b07b", True),
-            (None, 6, 0, "", False),
-            (None, 6, 1, "Error occurred", False),
+            # Fail: Container does not exists
```

Comment:
```suggestion
            # Fail: Container does not exist
```
And it should be "Failure" (noun) if we want to be consistent with "Success" (noun).